hb_report/constants.py

```diff
@@ -74,12 +74,11 @@
 start_stop Configuration.validated..Starting.heartbeat|Corosync.Cluster.Engine|Executive.Service.RELEASE|Requesting.shutdown|Shutdown.complete
 """
 
-PACKAGES = """pacemaker libpacemaker3
-pacemaker-pygui pacemaker-pymgmt pymgmt-client
-openais libopenais2 libopenais3 corosync libcorosync4
+PACKAGES = """pacemaker libpacemaker3 pacemaker-cli pacemaker-remote
+corosync libcorosync4
```

Comment:
actually libcorosync4 was splited by opensuse review team [1] like:
libcfg6
libcmap4
libcorosync_common4
libcpg4
libquorum5
libsam4
libtotem_pg5
libvotequorum8

and we also have new packages:
corosync-qdevice
corosync-qnetd

[1]https://build.opensuse.org/request/show/523762
@krig @liangxin1300 I think you should also add these packages as well